### PR TITLE
Added Features API

### PR DESCRIPTION
**Pull request by midithaf**

### CODE REVIEW

File: tests/endpoints/v1/test_features.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import unittest
+
+from predicthq.endpoints.v1.features.schemas import FeatureResultSet
+from tests import with_mock_client, with_mock_responses, with_client
+
+
+class FeaturesTest(unittest.TestCase):
+
+    @with_mock_client()
+    def test_request_params_dicts(self, client):
```
Comment (mbonnefoy) [inline tests/endpoints/v1/test_features.py:10]: The endpoints tests are used as "kitchen sinks" for confirming that all params are properly passed to the client. Do you have all the params here?

Any reason for not adding the underscores version?

**Review by mbonnefoy — APPROVED** [review]
Looks good @midithaf , just added a minor comment on the tests.

**Comment by driadi** [comment]
@midithaf we want to update the README too

**Comment by mbonnefoy** [comment]
> @midithaf we want to update the README too

you might also want to go the extra mile and add a couple of usecases in the usecases/ dir.

File: predicthq/endpoints/v1/features/schemas.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        serialize_when_none = True
+
+    stats = ListType(StringType(choices=('avg', 'count', 'max', 'median', 'min', 'sum', 'std_dev'), default='count'))
+    phq_rank = ModelType(IntRange)
+
+
+class FeatureGeoPoint(Model):
+    lat = FloatType(required=True)
+    lon = FloatType(required=True)
+    radius = StringType(regex=r'\d+(k?m|mi)', required=True, messages={
```
[the remark this answers is not in the document]
Comment (CockyAmoeba) [reply predicthq/endpoints/v1/features/schemas.py:19]: Should there not be a `required` flag set fro all 3 of these? also should there not be a validator assigned to to verify `lat` and alon`

**Review by CockyAmoeba — APPROVED** [review]

File: tests/endpoints/v1/test_features.py (hunk 1 above)
Comment (midithaf) [reply tests/endpoints/v1/test_features.py]: Added more tests with all params and  underscores version

**Pull request merged**